streamclient/streamclient_test.go

```diff
@@ -118,8 +119,9 @@ func TestNewConsumerAndProducer_StandardstreamConsumerAndKafkaProducer(t *testin
 
 	// Set the streamclient file descriptor to a temporary file, simulating
 	// received data in the Stdin fd.
-	options := func(s *standardstream.Client, _ *kafka.Client) {
+	options := func(s *standardstream.Client, kc *kafka.Client) {
 		s.ConsumerFD = f
+		kc.ConsumerConfig.Consumer.Offsets.Initial = sarama.OffsetNewest
```

Comment:
Maybe assert that this is correctly set?

Reply:
Hmm, there isn't really an interface back to these settings.

Reply:
Hmm too bad, well, at least this test proves we can set it